src/Resources/views/CRUD/edit_mongo_collection.html.twig

```diff
@@ -132,7 +132,7 @@ file that was distributed with this source code.
             {% endif %}
 
             {# include association code #}
-            {% include 'SonataDoctrineMongoDBAdminBundle:CRUD:edit_mongo_one_association_script.html.twig' %}
+            {% include '@SonataDoctrineMongoDBAdmin/CRUD/edit_mongo_one_association_script.html.twig' %}
```

Comment:
The twig dependency is missing in `composer.json`. Please add it

Reply:
@core23 done